android/app/build.gradle

```diff
@@ -84,12 +84,15 @@ project.ext.react = [
 apply from: "../../node_modules/react-native/react.gradle"
 apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
 
-if (System.getenv("SENTRY_ENABLED") == "true") {
-    project.ext.sentryCli = [
+
+/** Disable the below if you are not using Sentry */
+def SENTRY_ENABLED = true
```

Comment:
This should be set to an environment variable

Reply:
reverted